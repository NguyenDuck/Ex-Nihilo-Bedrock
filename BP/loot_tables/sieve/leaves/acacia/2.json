{
    "pools": [
        {
            "rolls": 1,
            "entries": [
                {
                    "type": "loot_table",
                    "name": "loot_tables/sieve/leaves/acacia/1.json"
                }
            ]
        },
        {
            "rolls": 1,
            "entries": [
                {
                    "type": "item",
                    "name": "sapling",
                    "functions": [
                        {
                            "function": "set_data",
                            "data": 4
                        }
                    ],
                    "weight": 10
                },
                {
                    "type": "empty",
                    "weight": 90
                }
            ]
        }
    ]
}