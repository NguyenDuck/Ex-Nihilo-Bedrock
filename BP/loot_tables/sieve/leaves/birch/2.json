{
    "pools": [
        {
            "rolls": 1,
            "entries": [
                {
                    "type": "loot_table",
                    "name": "loot_tables/sieve/leaves/birch/1.json"
                }
            ]
        },
        {
            "rolls": 1,
            "entries": [
                {
                    "type": "item",
                    "name": "sapling",
                    "functions": [
                        {
                            "function": "set_data",
                            "data": 2
                        }
                    ],
                    "weight": 10
                },
                {
                    "type": "empty",
                    "weight": 90
                }
            ]
        }
    ]
}