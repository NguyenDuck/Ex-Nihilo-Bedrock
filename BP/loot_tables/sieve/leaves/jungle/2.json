{
    "pools": [
        {
            "rolls": 1,
            "entries": [
                {
                    "type": "loot_table",
                    "name": "loot_tables/sieve/leaves/jungle/1.json"
                }
            ]
        },
        {
            "rolls": 1,
            "entries": [
                {
                    "type": "item",
                    "name": "sapling",
                    "functions": [
                        {
                            "function": "set_data",
                            "data": 3
                        }
                    ],
                    "weight": 10
                },
                {
                    "type": "empty",
                    "weight": 90
                }
            ]
        }
    ]
}