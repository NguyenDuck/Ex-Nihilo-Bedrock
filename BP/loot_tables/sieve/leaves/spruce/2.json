{
    "pools": [
        {
            "rolls": 1,
            "entries": [
                {
                    "type": "loot_table",
                    "name": "loot_tables/sieve/leaves/spruce/1.json"
                }
            ]
        },
        {
            "rolls": 1,
            "entries": [
                {
                    "type": "item",
                    "name": "sapling",
                    "functions": [
                        {
                            "function": "set_data",
                            "data": 1
                        }
                    ],
                    "weight": 10
                },
                {
                    "type": "empty",
                    "weight": 90
                }
            ]
        }
    ]
}