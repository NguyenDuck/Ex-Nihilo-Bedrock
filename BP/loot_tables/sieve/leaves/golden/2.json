{
    "pools": [
        {
            "rolls": 1,
            "entries": [
                {
                    "type": "loot_table",
                    "name": "loot_tables/sieve/leaves/golden/1.json"
                }
            ]
        },
        {
            "rolls": 1,
            "entries": [
                {
                    "type": "item",
                    "name": "apple",
                    "weight": 10
                },
                {
                    "type": "empty",
                    "weight": 90
                }
            ]
        },
        {
            "rolls": 1,
            "entries": [
                {
                    "type": "item",
                    "name": "golden_apple",
                    "weight": 0.3
                },
                {
                    "type": "empty",
                    "weight": 99.7
                }
            ]
        },
        {
            "rolls": 1,
            "entries": [
                {
                    "type": "item",
                    "name": "exnihilo:silkworm",
                    "weight": 5
                },
                {
                    "type": "empty",
                    "weight": 95
                }
            ]
        }
    ]
}